aries-site/src/layouts/main/Layout.js

```diff
@@ -53,58 +58,85 @@ export const Layout = ({
-                      <ContentSection>
-                        <DocsPageHeader
-                          title={title}
-                          topic={topic}
-                          render={render}
-                        />
-                        {children}
-                      </ContentSection>
-                      {relatedContent.length > 0 && (
-                        <RelatedContent
-                          relatedContent={relatedContent}
-                          title={title}
-                        />
+              <Head title={title} />
+              <Meta
+                title={title}
+                render={render}
+                description={seoDescription}
+                canonicalUrl={`https://design-system.hpe.design${router.route}`}
+              />
+              <>
+                <SkipLinks id="skip-links">
+                  <SkipLink id="main" label="Main Content" />
+                </SkipLinks>
+                <PageContent>
+                  <Header fill="horizontal" alignSelf="center" />
+                </PageContent>
+                <MainContentWrapper>
+                  <Main overflow="visible">
+                    <SkipLinkTarget id="main" />
+                    <Box direction={layout !== 'plain' ? 'row' : 'column'}>
+                      {layout !== 'plain' ? (
+                        <>
+                          <Box
+                            width={
+                              !['xsmall', 'small'].includes(size)
+                                ? 'calc(100% - 192px)'
+                                : undefined
+                            }
+                            pad={
+                              !['xsmall', 'small'].includes(size)
+                                ? { right: 'medium' }
+                                : undefined
+                            }
+                            margin={
+                              !['xsmall', 'small'].includes(size)
+                                ? { right: 'large' }
+                                : undefined
+                            }
+                          >
+                            <ContentSection>
+                              <DocsPageHeader
+                                title={title}
+                                topic={topic}
+                                render={render}
+                              />
+                              {children}
+                            </ContentSection>
+                            {relatedContent.length > 0 && (
+                              <RelatedContent
+                                relatedContent={relatedContent}
+                                title={title}
+                              />
+                            )}
+                            <FeedbackSection />
+                          </Box>
+                          <Box pad="small">
+                            {!['xsmall', 'small'].includes(size) ? (
```

Comment:
We should move the logic outside of the Box on line 123 so that we don't render an empty Box to the DOM.